kong/db/dao/plugins.lua

```diff
@@ -148,6 +149,9 @@ local function load_plugin_handler(plugin)
   local ok, handler = utils.load_module_if_exists(plugin_handler)
   if not ok and go.is_on() then
       ok, handler = go.load_plugin(plugin)
+      if type(handler) == "table" then
+        handler._go = true
```

Comment:
Maybe I don't understand this completely, but wouldn't moving `go_plugins_cnt = go_plugins_cnt + 1` be more intuitive?